### PR TITLE
Adds support for Wagtail 2.0 and Django 2.0 

### PR DESCRIPTION
**Pull request by ababic**
Tests are currently passing against the beta release with both Django 1.11 and 2.0.

I don't envisage there being any changes between now and the official release, but I'd like to wait and be sure before merging.

It's unknown at this time whether wagtail-condensedinlinepanel will be updated to support 2.0 any time soon, so I might add a note about that to the installation instructions where it is recommended.

### CODE REVIEW

File: wagtailmenus/models/menuitems.py [hunk 1]
```diff
@@ -28,7 +31,7 @@ class AbstractMenuItem(models.Model, MenuItem):
     """A model class that defines a base set of fields and methods for all
     'menu item' models."""
     link_page = models.ForeignKey(
-        'wagtailcore.Page',
```
Comment (Rotund) [inline wagtailmenus/models/menuitems.py]: Was this change needed? I believe the appname didn't actually change.

Is there a good reason to change away from lazy loading?

File: wagtailmenus/tests/test_backend.py [hunk 2]
```diff
@@ -186,22 +191,9 @@ def test_mainmenu_edit(self):
...
-
-    @modify_settings(INSTALLED_APPS={'append': 'condensedinlinepanel'})
-    def test_condensedinlinepanel(self):
-        from condensedinlinepanel.edit_handlers import CondensedInlinePanel
-        self.assertTrue(isinstance(FlatMenuItemsInlinePanel(), CondensedInlinePanel))
-        self.assertTrue(isinstance(MainMenuItemsInlinePanel(), CondensedInlinePanel))
-
     def test_mainmenu_edit_multisite(self):
         Site.objects.create(
-            hostname='test2.com', port=80, root_page_id=2,
```
Comment (Rotund) [inline wagtailmenus/tests/test_backend.py]: Shouldn't these lines still be there?

File: wagtailmenus/models/menuitems.py (hunk 1 above)
Comment (ababic) [reply wagtailmenus/models/menuitems.py]: Hi @Rotund. The model is imported at the top of the file, so I don't see any benefit from lazy loading in this case. I also think it makes the code slightly clearer (we definitely mean the same 'Page' model that is imported and referenced elsewhere in the file)

File: wagtailmenus/tests/test_backend.py (hunk 2 above)
Comment (ababic) [reply wagtailmenus/tests/test_backend.py]: No. The condensedinlinepanel tests are moved further down, and the extra sites being created in the other test are unnecessary.

**Comment by StriveForBest** [comment]
👍 
I think the readme needs to be updated too. Very needed to have django 2/wagtail 2 support!
